README3.adoc

```diff
@@ -1111,6 +1137,84 @@ class MyApplication : Application {
 
 CAUTION: Objects that are bound with `autoActivitySingleton` will always be injected according to *the last displayed activity*.
 
+==== The service scope
+
+If you want a singleton that lives only during the lifecycle of a specific `Service`, and not any `Context`, you can use the activity scope.
```

Comment:
Small copy-paste typo here, I think you mean "you can use the **service** scope" ;)